src/settings-ui/Settings.UI/SettingsXAML/Styles/InfoBadge.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <Style x:Key="NewInfoBadge" TargetType="InfoBadge">
+        <Setter Property="Padding" Value="5,0,5,0" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="InfoBadge">
+                    <Grid
+                        x:Name="RootGrid"
+                        Padding="{TemplateBinding Padding}"
+                        Background="{TemplateBinding Background}"
+                        CornerRadius="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.InfoBadgeCornerRadius}">
+                        <TextBlock
+                            x:Uid="SettingsPage_NewInfoBadge"
+                            HorizontalAlignment="Center"
+                            VerticalAlignment="Center"
+                            FontSize="{ThemeResource InfoBadgeValueFontSize}"
+                            FontWeight="SemiBold" />
```

Comment:
Why semibold? Can't seem to find that in the default style. Also not in line with general guideline for typography. (given that the font-size is only 11)